pkg/server/status_test.go

```diff
@@ -1127,6 +1127,59 @@ func TestHotRanges2Response(t *testing.T) {
 	}
 }
 
+func TestNetworkConnectivity(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+	numNodes := 3
+	testCluster := serverutils.StartNewTestCluster(t, numNodes, base.TestClusterArgs{
+		ServerArgs: base.TestServerArgs{
+			Knobs: base.TestingKnobs{
+				Store: &kvserver.StoreTestingKnobs{
+					ReplicaPlannerKnobs: plan.ReplicaPlannerTestingKnobs{
+						DisableReplicaRebalancing: true,
```

Comment:
Do you want just this?

https://github.com/cockroachdb/cockroach/blob/42e45b4b8160f8a550a298b64f43799072fdfb5a/pkg/kv/kvserver/client_lease_test.go#L113-L119

The setup here seems bespoke. The snippet above will disable all queues, so all replicas will be only on n1, which seems appropriate for this test.

Reply:
Thanks, that's what I needed!